README.md

```diff
@@ -5,44 +5,44 @@ Our open source community's Wiki focuses on answering your questions about learn
 - Curriculum
 - Other Programming Languages
 - Local Campsite Communities
-- Nonprofit Projects  
+- Nonprofit Projects
  ...and much more
```

Comment:
@Rafase282 Adding two spaces at the end of line is same as using `<br>` in *gfmd*.